### PR TITLE
[3.x] Add missing property types and return types for PHP 8.1

### PR DESCRIPTION
**Pull request by danielrona**
This pull should fix a couple of bugs and implement PHP8.1 for the most, so far I've not run into any issues as of yet 

- yajra/laravel-oci8#692 
- yajra/pdo-via-oci8#87
- yajra/laravel-oci8#695

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     ],
     "require": {
         "php": "^8.0",
-        "ext-pdo": "*"
+        "ext-pdo": "*",
+        "ext-oci8": "*"
```
Comment (yajra) [inline composer.json:14]: This was recently requested to be removed by some users. Please remove, thanks!

**Review by yajra — CHANGES_REQUESTED** [review]

**Comment by yajra** [comment]
I've fixed and added a PHP8.1 github action on master branch. Can you please rebase on master and check if tests are working? I cannot update to PHP8 & 8.1 yet so I can't check this thoroughly. Thanks!

File: composer.json (hunk 1 above)
Comment (yajra) [reply composer.json:14]: Nvm this review. Let's have this enforced. Thanks!

**Comment by yajra** [comment]
@danielrona changes looks good and tests are passing.

Requesting review from other users. If no response in a week or two, will merge this. Thanks!

**Review by yajra — APPROVED** [review]

**Comment by X-Coder264** [comment]
@yajra Any plans to merge and tag a release with this soon? We've just deployed Laravel 9 + PHP 8.1 to prod and our Sentry is full of these deprecation messages now.

**Comment by yajra** [comment]
Unfortunately, can't upgrade yet. if you can confirm that this is working then I guess I can release this asap.

**Comment by X-Coder264** [comment]
I can do some basic testing over the weekend and let you know if my code still works with these changes.

**Comment by robsontenorio** [comment]
@X-Coder264 what branch do you use for Laravel 9 + PHP 8.1 ?

**Comment by yajra** [comment]
Sorry for late checking, I am now finally able to use 8.1 and can confirm that deprecate warning were fixed. Will be included in the next release.

**Comment by yajra** [comment]
Released on v3.2.1, thanks! 🚀

**Pull request merged**